util/formula_templater/config.hcl

```diff
@@ -444,9 +444,9 @@ formula {
 
 formula {
     product = "tf-migrate"
-    name = "tf-migrate"
-    desc = "tf-migrate"
-    homepage = "https://www.terraform.io"
+    name = "TfMigrate"
+    desc = "Terraform Migrate"
+    homepage = "https://github.com/hashicorp/tf-migrate"
```

Comment:
I don't see why the homepage needs to be updated to fix the class name.  The repo is private, so the homepage link will be broken for everyone not in HashiCorp.  Did the tf-migrate team request this?

Reply:
I completely misread the repo name and thought this was internal. Fixing.